data_curation/augmentation.py

```diff
@@ -1,7 +1,39 @@
 import random
 import re
 import typing
-from typing import Tuple
+
+import pydantic
+
+import data_curation.schemas.augmentation as schemas
+from utils import normalize as normalize
+from utils import path as path
+
+# TODO: move these loads to path or normalize module
+LOINC_METHOD_ENHANCEMENTS = path.read_json(
+    "data/snoinc_extracts/loinc_method_abbrv_syn_20250916.json"
+)
+LOINC_PROPERTY_ENHANCEMENTS = path.read_json(
+    "data/snoinc_extracts/loinc_property_abbrv_syn_20250916.json"
+)
+LOINC_SCALE_ENHANCEMENTS = path.read_json(
+    "data/snoinc_extracts/loinc_scale_abbrv_syn_20250916.json"
```

Comment:
Just realized that the files are still not completely static.  I just did an update to filter out some 'bad' results from this data.  Maybe we could change this to get the 'latest' file?  This may be a bit more complex....if anything, maybe we change this to be the files dated as of today (from my PR that 'should' be out shortly).